Source/WebKit/Configurations/webpushd.xcconfig

```diff
@@ -41,8 +41,11 @@ CODE_SIGN_ENTITLEMENTS[sdk=*simulator] = Resources/ios/XPCService-embedded-simul
 
 WK_PROCESSED_XCENT_FILE=$(TEMP_FILE_DIR)/$(FULL_PRODUCT_NAME).entitlements
 
+WK_LIBRARY_VALIDATION_CODE_SIGN_FLAGS = $(WK_LIBRARY_VALIDATION_CODE_SIGN_FLAGS_$(WK_LIBRARY_VALIDATION_ENABLED))
```

Comment:
Where does `WK_LIBRARY_VALIDATION_ENABLED` ever get enabled? For XPC Services, it looks like `BaseXPCService.xcconfig` would define that as YES if it starts out as empty: https://github.com/WebKit/WebKit/blob/45e2f437b91c9aaeb1aa984bb6e4404e9f3fddac/Source/WebKit/Configurations/BaseXPCService.xcconfig#L96

but it's not clear to me where this would come from for this target, which isn't an XPC service.

Reply:
It's disabled by default, and only enabled on production builds via other build system config changes.

Reply:
Yeah I was under the mistaken presumption that we set `WK_LIBRARY_VALIDATION_ENABLED` externally from as a project override in e.g. projects.json, but I guess you are right in that we just default library validation to enabled unless it's forced off via those lines in `BaseXPCService.xcconfig`. So I need to move that logic up to a more base xcconfig.